packages/babel-helpers/src/helpers/getPrototypeOf.ts

```diff
@@ -0,0 +1,16 @@
+/* @minVersion 7.0.0-beta.0 */
+
+type Object = {
+  __proto__?: any;
+} & { [key: string]: unknown };
+
+export default function _getPrototypeOf<T extends Object>(o: T) {
+  // @ts-expect-error explicitly assign to function
+  _getPrototypeOf = Object.setPrototypeOf
+    ? // @ts-expect-error -- intentionally omitting the argument
+      Object.getPrototypeOf.bind(/* undefined */)
+    : function _getPrototypeOf<T extends Object>(o: T) {
+        return o.__proto__ || Object.getPrototypeOf(o);
```

Comment:
Instead of defining `Object` with `__proto__`, we can use `o["__proto__"]` and TS should stop complaining.

Ad an alternative, we can use
```ts
interface Object {
    __proto__: object | null;
}
```

and then TS will automatically let us use `__proto__` on things of type `object` (with the lowercase `o`)

Reply:
Merging with `interface Object` is bad. If we do this, TypeScript will add `__proto__` to **every object** out there. Let's just use `(as any).__proto__`, it is better than `__proto__` goes rouge.